help/user-guide/create-sample-isajson-compliant.md

```diff
@@ -3,59 +3,67 @@ title: SEEK User Guide - Create Sample in ISA-JSON compliant Experiments
 layout: page
 ---
 
-# Viewing samples in ISA-JSON compliant Experiments
-Samples in ISA-JSON compliant Experiments can be visualised in the "design" tab (Study or Assay design). Samples are grouped in tables: Study Sources table, Study Samples table, Assay Samples table, Experiment overview.
+# Working with Samples in ISA-JSON compliant Experiments
 
-## Sources table and Samples table
-Sources table and Samples table are interactive tables (dynamic tables) that allow samples creation and editing. 
-* Studies contain both Sources table and Samples table.
-* Assays contains only Samples table.
+## Types of samples in ISA-JSON compliant Experiments
 
-## samples
```

Comment:
@somathias why do you suggest to delete these paragraphs?

Reply:
Nothing is deleted, only moved.

Reply:
ok